Runtime/GuiSys/CHudVisorBeamMenu.cpp

```diff
@@ -10,22 +12,40 @@
 
 namespace urde {
 
-static const char* BaseMenuNames[] = {"BaseWidget_VisorMenu", "BaseWidget_BeamMenu"};
+constexpr std::array BaseMenuNames{
```

Comment:
these could be std::array<std::string_view>, or is that what it becomes automatically?

Reply:
These become `std::array<const char*, N>`, but there's no real difference in this scenario if either type is used (aside from `const char*` using less space in the resulting binary).